test/test_irods_http_api.py

```diff
@@ -2653,6 +2653,74 @@ def test_modifying_metadata_atomically(self):
         self.assertEqual(result['irods_response']['status_code'], 0)
         self.assertEqual(len(result['rows']), 0)
 
+    def test_modifying_resource_properties(self):
+        headers = {'Authorization': 'Bearer ' + self.rodsadmin_bearer_token}
+        resource = 'test_modifying_resource_properties_original'
+
+        # Create a new resource.
+        r = requests.post(self.url_endpoint, headers=headers, data={
+            'op': 'create',
+            'name': resource,
+            'type': 'replication'
+        })
+        self.logger.debug(r.content)
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.json()['irods_response']['status_code'], 0)
+
+        try:
+            # The list of updates to apply in sequence.
+            property_map = [
+                ('name',        'test_modifying_resource_properties_renamed'),
+                ('type',        'passthru'),
+                ('host',        'example.org'),
+                ('vault_path',  '/tmp/test_modifying_resource_properties_vault'),
+                ('status',      'down'),
+                ('status',      'up'),
```

Comment:
Might be worth adding a test with an invalid value here since these are being checked in the HTTP API. Thoughts?

Reply:
Done.